templates/row-accordion-section-template.html

```diff
@@ -0,0 +1,14 @@
+<div class="row"> <!-- Start  ACCORDION_ID row -->
+    <div class="accordion" id="{{ACCORDION_ID}}">
+      <div class="accordion-item"> <!-- Start ACCORDION_ID item -->
+        <h2 class="accordion-header" id="{{ACCORDION_HEADER_ID}}">
+          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#{{ACCORDION_COLLAPSE_ID}}" aria-expanded="false" aria-controls="{{ACCORDION_COLLAPSE_ID}}">
```

Comment:
I cannot re-collapse the accordion once it's been opened by clicking this button

Reply:
Don't think it would work in the template version.  Once it's brought in to the main page (eg. index page) it does work.  If it doesn't then there's  bootstrap conflict.

Reply:
yeah it doesn't work on the index page. Leave this for now and we can fix it in a later PR. I'll make an issue for it

Reply:
See #4